pkg/cpi/commonUtils.go

```diff
@@ -62,3 +62,14 @@ func (tokenParameters TokenParameters) GetBearerToken() (string, error) {
 	token := jsonResponse.Path("access_token").Data().(string)
 	return token, nil
 }
+
+//CloseResponseBodyIfNecessary - close the http response object
+func CloseResponseBodyIfNecessary(response *http.Response) {
```

Comment:
The idea was to use it in two places of the code. Once for the token response and once for the [deploy response](https://github.com/SAP/jenkins-library/pull/2474/files/7f82052890145b7f7904d118196861961ace3042..366c54e64f2dc2cff44bc732208f8922b8b3232b#diff-63bf7a3c4709ee705740f814bdcc6ca2374d108181448211cd745806bdab27b9R78).